crates/re_arrow_store/src/store_read.rs

```diff
@@ -750,7 +751,7 @@ impl IndexedBucket {
                     %primary,
                     ?components,
                     timeline = %self.timeline.name(),
-                    time = self.timeline.typ().format(time),
+                    time = self.timeline.typ().format(time, TimeZone::Utc),
```

Comment:
This is such a common pattern that I would suggest we add `format_utc`:

```suggestion
                    time = self.timeline.typ().format_utc(time),
```

Reply:
Sure, done!